tests/services/repo-creator.service.spec.ts

```diff
@@ -26,4 +36,54 @@ describe('RepoCreatorService', () => {
       expect(githubService.isRepositoryPresent).toHaveBeenCalledWith(PHASE_OWNER, PHASE_REPO);
     });
   });
+
+  describe('.attemptRepoCreation()', () => {
+    it('should not need to create the repository if no permissions were needed', () => {
```

Comment:
```suggestion
    it('should not create the repository if repo creation is not needed', () => {
```
It seems a bit clearer to say that "repo creation is not needed" (rather than "permission is not needed")

Reply:
Yea this makes it more clearer